ysoserial/Helpers/DevTest/SerializersHelper.cs

```diff
@@ -1,26 +1,19 @@
-﻿using Microsoft.VisualStudio.Text.Formatting;
-using System;
-using System.Collections;
-using System.Collections.Generic;
+﻿using System;
 using System.Collections.Specialized;
-using System.ComponentModel.Design;
-using System.Data;
 using System.Diagnostics;
 using System.Globalization;
 using System.IO;
-using System.Linq;
 using System.Reflection;
 using System.Runtime.Serialization;
-using System.Runtime.Serialization.Formatters.Binary;
 using System.Text;
-using System.Web.UI.WebControls;
 using System.Windows;
 using System.Windows.Data;
 using System.Windows.Markup;
 using System.Xml;
 using System.Xml.Serialization;
 
-namespace ysoserial.DevTest
+// Coded by Soroush Dalili (@irsdl)
```

Comment:
We dont add who coded what, if anyone is interested, they can review git history